talk/owt/sdk/base/functionalobserver.h

```diff
@@ -20,7 +21,7 @@ class FunctionalCreateSessionDescriptionObserver
       std::function<void(webrtc::SessionDescriptionInterface*)> on_success,
       std::function<void(const std::string&)> on_failure);
   virtual void OnSuccess(webrtc::SessionDescriptionInterface* desc);
-  virtual void OnFailure(const std::string& error);
+  virtual void OnFailure(webrtc::RTCError error);
```

Comment:
`const webrtc::RTCError&`?

Reply:
no. the base class for this callback is using webrtc::RTCError not the const & version.

Reply:
Okay.